openbb_terminal/stocks/stocks_helper.py

```diff
@@ -1108,3 +1073,45 @@ def heikin_ashi(data: pd.DataFrame) -> pd.DataFrame:
     ]
 
     return pd.concat([data, ha], axis=1)
+
+
+def calculate_adjusted_prices(df: pd.DataFrame, column: str, dividends: bool = False):
```

Comment:
This helper was committed unintentionally and is part of fixing something unrelated.  It can stay or go, but can be used to help fix some issues with the AlphaVantage adjusted prices data.